gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/AliasBasedTokenStateService.java

```diff
@@ -104,14 +104,15 @@ public long getTokenExpiration(final String token) {
 
   @Override
   public void revokeToken(final String token) {
-    // Record the revocation by setting the expiration to -1
-    updateExpiration(token, -1L);
+    /* no reason to keep revoked tokens around */
+    removeRevokedExpiredToken(token);
     log.revokedToken(getTokenDisplayText(token));
   }
 
   @Override
   protected boolean isRevoked(final String token) {
```

Comment:
I'm wondering if we need this method at all since there is no need to distinguish revoked tokens from unknown tokens.

Reply:
Sure, updated.